pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -1801,7 +1801,7 @@
       "BaselineVersion": "4.0.0",
       "AssemblyVersionInPackageVersion": {
         "4.0.0.0": "4.0.0",
-        "4.0.1.0": "4.3.0"
+        "4.1.0.0": "4.3.0"
```

Comment:
This change was produced automatically after I updated dir.props and then built the package with /t:UpdatePackageIndex.  Is there something I should be doing differently?